File: api/code-samples/python/debug/profiling/writeMutexProfile.py
import json
from opensdk.sdk import OpenSDK

host = "https://api.baobab.klaytn.net:8651"

fileName = "mutex.profile"

sdk = OpenSDK(host)
debug_response = sdk.debug.write_mutex_profile(fileName)

print(json.loads(debug_response.response.data))

File: web3rpc/rpc-specs/code-samples/javascript/debug/profilling/writeMutexProfile.js
const { JsonRpcProvider } = require("@klaytn/ethers-ext");
(() => {
let sdk = new JsonRpcProvider("https://public-en-baobab.klaytn.net");    
    const file = "mutex.profile";

    sdk.debug.writeMutexProfile(file, {}, (err, data, response) => {
        console.log(data);
    });
}
)()